apps/daffio/src/app/core/sidebar/containers/sidebar-viewport/sidebar-viewport.component.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Store, StoreModule, combineReducers } from '@ngrx/store';
+
+import { DaffioSidebarViewportContainer } from './sidebar-viewport.component';
+import * as fromSidebar from '../../reducers/index';
+import { ToggleSidebar, OpenSidebar, CloseSidebar, SetSidebarState } from '../../actions/sidebar.actions';
+import { DaffSidebarModule, DaffSidebarViewportComponent } from '@daffodil/design';
+import { By } from '@angular/platform-browser';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { Component, Output, EventEmitter } from '@angular/core';
+
+@Component({selector: 'daffio-sidebar', template: ''})
+class MockDaffioSidebarContainer {
+  @Output() close: EventEmitter<any> = new EventEmitter();
+}
+
+describe('DaffioSidebarContainer', () => {
+  let component: DaffioSidebarViewportContainer;
+  let fixture: ComponentFixture<DaffioSidebarViewportContainer>;
+  
+  let sidebarViewport: DaffSidebarViewportComponent;
+
+  let store: Store<fromSidebar.State>;
+  let stubShowSidebar: boolean;
+  let sidebarContainer: MockDaffioSidebarContainer;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        StoreModule.forRoot({
+          daffioSidebar: combineReducers(fromSidebar.reducers),
+        }),
+        NoopAnimationsModule,
+        DaffSidebarModule,
+      ],
+      declarations: [ 
+        DaffioSidebarViewportContainer,
+        MockDaffioSidebarContainer
+      ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DaffioSidebarViewportContainer);
+    component = fixture.componentInstance;
+    store = TestBed.get(Store);
+    spyOn(store, 'dispatch');
+
+    fixture.detectChanges();
+
+    sidebarViewport = fixture.debugElement.query(By.css("daff-sidebar-viewport")).componentInstance;
+    sidebarContainer = fixture.debugElement.query(By.css("daffio-sidebar")).componentInstance;
```

Comment:
resolved in cfc6f56

Reply:
I think you misunderstood what I meant. Though you did bring to my attention another issue (I'll add another comment). I mean that these two names (on lines 52-53) should be "daffSidebarViewport" and "daffioSidebar". Otherwise it's difficult to tell what they are later in the tests

Reply:
Resolved in f4dff39.